Packages/eu.netherlands3d.indicators/Runtime/Scripts/DossierSO.cs

```diff
@@ -137,11 +154,67 @@ public IEnumerator LoadMapDataAsync(Frame frame)
             frame.mapData = mapData;             
         }
 
+        public IEnumerator Import(string sourceIri)
+        {
+            dossierSystemState = DossierSystemState.Importing;
+            onImporting.Invoke();
+            string url = AssembleImportUri();
+            Debug.Log($"<color=orange>Importing dossier with iri {sourceIri} from {url}</color>");
+
+            var www = PostAsJson(url, new ImportRequest { iri = sourceIri });
+            yield return www.SendWebRequest();
+
+            if (www.result != UnityWebRequest.Result.Success)
+            {
+                onImportingFailed.Invoke();
+                dossierSystemState = DossierSystemState.ImportingFailed;
+                Debug.Log($"<color=red>Failed to import dossier from {sourceIri}</color>");
+                Debug.LogError(www.error);
+                yield break;
+            }
+
+            var json = www.downloadHandler.text;
+            Debug.Log($"<color=orange>Received response data when importing dossier</color>");
+            Debug.Log(json);
+            
+            Dossier dossier;
+            try
+            {
+                dossier = JsonConvert.DeserializeObject<Dossier>(json);
+            }
+            catch (Exception e)
+            {
+                Debug.Log($"<color=red>Failed to deserialize dossier while importing</color>");
+                Debug.LogError(e.Message);
+                dossierSystemState = DossierSystemState.ImportingFailed;
+                onImportingFailed.Invoke();
+                yield break;
+            }
```

Comment:
Ik raad aan geen try/catch te gebruiken.
Exceptions aan hebben staan geeft heel veel performance overhead bij WebGL builds. 
Voor nu hebben we die aan staan bij de twin, maar het zou fijn zijn als we die op den duur op [WebGLExceptionSupport.None](https://docs.unity3d.com/ScriptReference/WebGLExceptionSupport.html) zouden kunnen zetten, net als bij 3D Amsterdam.
Deze try/catch zou dan dan de applicatie compleet laten freezen. (ipv. exceptions in de console krijg je dan een harde crash)